plugins/modules/packaging/os/rpm_ostree_pkg.py

```diff
@@ -32,9 +32,17 @@
       choices: [ 'absent', 'present' ]
       default: 'present'
       type: str
+    apply_live:
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.